### PR TITLE
[W6.4b][W15-B3]See Jia Min Jasmine 

### PR DESCRIPTION
**Pull request by JasmineSee**
Added more non-functional requirements to Developer Guide

### CODE REVIEW

File: doc/DeveloperGuide.md [hunk 1]
```diff
@@ -94,6 +94,9 @@ Use case ends.
 2. Should be able to hold up to 1000 persons.
 3. Should come with automated unit tests and open source code.
 4. Should favor DOS style commands over Unix-style commands.
+5. Should work on any platforms that can support graphical user interface.
```
Comment (okkhoy) [inline doc/DeveloperGuide.md:97]: better to list a few platforms

File: doc/DeveloperGuide.md [hunk 2]
```diff
@@ -94,6 +94,9 @@ Use case ends.
...
+6. Should be backward compatible with data produced by earlier versions of the system.
```
Comment (okkhoy) [inline doc/DeveloperGuide.md:98]: which system? please try to be specific.

**Comment by okkhoy** [comment]
@JasmineSee 
Some comments added. Please ack & close the PR after reading comments.

**Comment by JasmineSee** [comment]
Noted and acknowledged. Thanks for the comments.

**Pull request closed without merging**